examples/server_events.rs

```diff
@@ -30,6 +32,21 @@ fn cron_event_handler(_ctx: &Context, _hz: u64) {
     NUM_CRONS.fetch_add(1, Ordering::SeqCst);
 }
 
+#[client_changed_event_handler]
+fn client_changed_event_handler(ctx: &Context, client_event: ClientChangeSubevent){
+    if let ClientChangeSubevent::Connected = client_event {
```

Comment:
We can also check and handle the disconnects in this example - by decrementing the counter for example